src/events.c

```diff
@@ -41,6 +45,37 @@ static void inform_keyboard_attached(gboolean value)
         }
 }
 
+static void dir_changed_cb(GFileMonitor *monitor,
+               GFile *file, GFile *other_file,
+               GFileMonitorEvent event_type, gpointer user_data) {
+    (void)monitor;
+    (void)file;
+    (void)other_file;
+    (void)event_type;
+    (void)user_data;
```

Comment:
There're some examples of unused variables in the sources. In all cases the unused variables don't have "(void)expression;"
```
static gboolean
kdbus_sock_read_cb(GIOChannel* ch, GIOCondition cond, gpointer not_used)
```
```
static gboolean
kdbus_sock_event_cb(GIOChannel *ch, GIOCondition cond, gpointer not_used)
```
`void handle_kevent(DBusMessage* m)`

All examples are taken from this package (ke-recv-extra)